src/app/customizer/GitHub-Profile-generator/page.jsx

```diff
@@ -0,0 +1,1052 @@
+                    alt="GitHub Langs"
+                  />
+                </div>
+              </div>
+            )}
+
+            {/* Support Me */}
+            {(formValues.buyMeACoffee || formValues.kofi) && (
+              <div className="mt-4">
+                <h1 className="text-2xl font-bold mt-8 mb-4">💖Support Me!</h1>
+                <div className="flex justify-center space-x-4">
+                  {formValues.buyMeACoffee && (
+                    <a
+                      href={formValues.buyMeACoffee}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      <img
+                        src="https://img.shields.io/badge/Buy_Me_A_Coffee-FBBC05?style=for-the-badge&logo=buy-me-a-coffee&logoColor=white"
+                        alt="Buy Me a Coffee Badge"
+                        className="h-6"
+                      />
+                    </a>
+                  )}
+
+                  {formValues.kofi && (
+                    <a
+                      href={formValues.kofi}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      <img
+                        src="https://img.shields.io/badge/Ko--fi-F16061?style=for-the-badge&logo=kofi&logoColor=white"
+                        alt="Ko-fi Badge"
+                        className="h-6"
+                      />
+                    </a>
+                  )}
+                </div>
+              </div>
+            )}
+          </div>
+        ) : (
+          <div className="mt-6 p-4 bg-gray-700 rounded-lg text-white">
+            <h3 className="text-lg font-semibold mb-2">Markdown Preview:</h3>
+
+            <div className="markdown-preview" />
+            <div
+              className="markdown-preview"
+              dangerouslySetInnerHTML={{ __html: markdownPreview }}
+            />
+
+            {/* {markdownPreview} */}
+
+            <div className="mt-2">
+              <button
+                onClick={handleCopyClick}
+                className="px-4 py-2 bg-blue-500 text-white rounded-lg hover:bg-blue-600"
+              >
+                Copy Markdown
+              </button>
+              {copySuccess && (
+                <p className="mt-2 text-green-500">{copySuccess}</p>
+              )}
+            </div>
+          </div>
+        )}
+      </div>
+
+      {/* Preview section end */}
+      <div className="flex justify-center mt-8">
+        <button
+          onClick={() => setView(view === "Preview" ? "Markdown" : "Preview")}
+          className={`${isDarkMode ? "bg-gray-800 text-gray-200" : "bg-blue-600 text-white"} p-3 rounded-lg`}
+        >
+          {view === "Preview"
+            ? "Switch to Markdown View"
+            : "Switch to Preview View"}
+        </button>
```

Comment:
Add a button to download it

Reply:
> Add a button to download it

is it necessary to add a download button?

Reply:
> > Add a button to download it
> 
> is it necessary to add a download button?

It can be a feature feat